setup.cfg

```diff
@@ -9,7 +9,7 @@ show-response = 1
 
 [pytest]
 minversion = 2.3.3
-norecursedirs = ".tox" "build" "docs[\/]_build" "astropy[\/]extern" "astropy[\/]utils[\/]compat"
+norecursedirs = ".tox" "build" "docs[\/]_build" "astropy[\/]extern" "astropy[\/]utils[\/]compat[\/]futures"
```

Comment:
This ensures that tests in `utils.compat.numpy` get run (by not excluding them any more). I checked that only `futures` had tests that do not work.